tests/unittests/test_file_accessor.py

```diff
@@ -2,12 +2,18 @@
 # Licensed under the MIT License.
 
 import os
+import sys
 import unittest
+from unittest import skipIf
+
 from azure_functions_worker import testutils
 from azure_functions_worker.bindings.shared_memory_data_transfer \
     import SharedMemoryException
 
 
+@skipIf(sys.platform == 'darwin', 'MacOS M1 machines do not correctly test the'
```

Comment:
nit: can we extract these as constants?

Reply:
Extract OS platform names as Constants?

Reply:
yep

Reply:
I'll skip it for now - as it might bring in some cyclic dependencies being caused due to adding a new Constant. Also, we want to solve and remove this in the future.